general/fonts-colors.css

```diff
@@ -0,0 +1,16 @@
+:root {
+  --primary-color: darkblue;
+  --secondary-color: lightblue;
+  /* --bkg-clr:linear-gradient(45deg,aqua  0%, rgba(182, 241, 252, 1) 50%, rgba(252, 174, 241, 1) 70%); */
+  --secondary-text-color: gray;
+  --hover-colour: lightgray;
+  --primary-text-color: black;
+  --danger-color: red;
```

Comment:
Be consistent in defining the colors, use hex or rgb, avoid using names.

Reply:
okay ...thank you